### PR TITLE
add wallet endpoints translation

### PR DESCRIPTION
**Pull request by keyganker**

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -292,7 +293,7 @@ en:
   ### User Trade Records
   wallet_heading_tradeRecords: User Trade Records
   wallet_para_tradeRecords: Get user's trading records. The last item is the oldest.
-  wallet_aside_tradeRecords: This endpoint will provide multiple trades if <code>order_id</code> is not provided.
+  wallet_aside_tradeRecords: One order may has multiple trade records.
```
Comment (keyganker) [inline locales/en.yml]: One order may also has multiple trade records

Comment (keyganker) [reply locales/en.yml]: @dexter-2

**Comment by keyganker** [comment]
@dexter-2

File: locales/en.yml [hunk 2]
```diff
@@ -286,13 +287,17 @@ en:
   ### Wallet Fund Records
   wallet_heading_walletRecords: Wallet Fund Records
   wallet_para_walletRecords: Get wallet fund records.
+  wallet_aside_walletRecords: You can find more detail of type <code>Withdraw</code> and <code>Refund</code> in <a href="#t-wallet_heading-t-account_heading_key-t-wallet_heading_withdrawrecords">Withdraw Records</a> endpoint.
   ### Withdraw Records
   wallet_heading_withdrawRecords: Withdraw Records
   wallet_para_withdrawRecords: Get withdrawal records.
+  wallet_aside_withdrawRecords: The difference between data in current endpoint and data of type <code>Withdraw</code> in <code>Wallet Fund Records</code> endpoint<span>:</span>
+    <p>For current endpoint, one withdrawal operation one record, and you can check current withdrawal state by field <code>status</code>.</p>
```
Comment (keyganker) [inline locales/en.yml]: Add note for wallet fund record endpoint and withdraw record endpoint

File: locales/en.yml [hunk 3]
```diff
@@ -286,13 +287,17 @@ en:
...
+    <p>For <code>Wallet Fund Records</code> endpoint，once user submited the withdrawal application, there will be a record whith type <code>Withdraw</code>, and if the application is <code>CancelByUser</code>, <code>Reject</code> or <code>Expire</code>, there will be another record with type <code>Refund</code>.</p>
```
Comment (keyganker) [inline locales/en.yml]: Add note for wallet fund record endpoint and withdraw record endpoint

File: source/includes/inverse_future/_wallet_data.md [hunk 4]
```diff
@@ -126,7 +134,7 @@ GET
     "ext_code": "",
     "ext_info": "",
     "result": {
-        "order_id": "",
+        "order_id": "t(:comment_abandoned)", // t(:comment_abandoned)
```
Comment (dextertd) [inline source/includes/inverse_future/_wallet_data.md:137]: What does this mean?

**Review by dextertd — CHANGES_REQUESTED** [review]
#13

File: locales/en.yml (hunk 3 above)
Comment (dextertd) [reply locales/en.yml]: I'm 100% sure I understand this... But I have tried to make it more clear in my review PR.

File: source/includes/inverse_future/_wallet_data.md (hunk 4 above)
Comment (keyganker) [reply source/includes/inverse_future/_wallet_data.md]: there is a historical reason:
Initially, this endpoint only support query by order_id, and so the response is :
```
{
order_id:xxx,
list:[
  trade item1
  trade item 2
]
}
```
and later it support query without order_id, so the order_id in the response is moved into the list:
```
{
order_id:"",//abandoned
list:[
  trade item1 with order_id1
  trade item 2 with order_id2
]
}

```

Comment (keyganker) [reply source/includes/inverse_future/_wallet_data.md:137]: > What does this mean?

there is a historical reason:
Initially, this endpoint only support query by order_id, and so the response is :
```
{
order_id:xxx,
list:[
  trade item1
  trade item 2
]
}
```
and later it support query without order_id, so the order_id in the response is moved into the list:
```
{
order_id:"",//abandoned
list:[
  trade item1 with order_id1
  trade item 2 with order_id2
]
}

```

**Pull request merged**